### ISSUE TITLE
Modifier Idea - Metal

### ISSUE THREAD

**Issue by Bear64DX** [EmeraldLoc/tag#319]
Basically, everyone would be encased in metal, it wouldn't matter whether you're a runner or not. The knockback could be slightly increased, but because you're encased in metal, you'd move just a little slower. You can still swim though. Basically, it's a combination of juggernaut, and incognito (feel free to decline this I just thought it was cool).

**Comment by EmeraldLoc** [comment]
> (feel free to decline this I just thought it was cool)
Oh I definitely will decline ideas.

This one's a neat idea! However making every player metal simply isn't viable, there'd need to be a way to make taggers and runners look different.

**Comment by TheMan6900** [comment]
what if runners color palettes were forced to be blue, while taggers were forced to be red.

**Comment by Bear64DX** [comment]
That's a good idea. The whole point is that everyone looks similar, which makes it hard to tell who is who. But if that's not okay, then you can simply make some of the colors different for each team.

**Comment by Bear64DX** [comment]
Hmm, that seems like the only option, however I think there's a better way to handle this rather than that. What if you made the metal texture look different for each team? Not talking about the color, but the actual design of the metal.

**Comment by EmeraldLoc** [comment]
Yea, that's more what I was thinking. I'll need to look into how the metal stuff works.

**Comment by Bear64DX** [comment]
Alright, I noticed that if you were to go into a MarioHunt Lobby, and you're in the lobby itself with the metal texture, you actually look different than the base game's metal. It's kinda hard to explain, but host up a MarioHunt server, make the hunters look metal, and make yourself metal. You'll notice that it looks different than the base game's metal texture.

**Comment by EmeraldLoc** [comment]
That's correct. The lobby changes the lighting direction. I can't do this without chaing everyone's lighting though. I can look into doing something like mh does with the glowing, which may work.

**Comment by Bear64DX** [comment]
I made a new issue, not only giving a suggestion for Tag, but as a suggestion for the metal modifier.

**Comment by Bear64DX** [comment]
Wait, in 2.4.1, there's a new metal texture. What if runners had the new metal texture, while taggers had the old metal texture?
